Sky Atlas/Sky Atlas.html

```diff
@@ -6344,7 +6344,7 @@ <h1>Sections & Primary Docs</h1>
           </td>
           <td>Near Term Process</td>
```

Comment:
```suggestion
          <td>Near-Term Process</td>
```

This suggestion should also be added on line 6343